### ISSUE TITLE
since 4.7.0: Wordpress is not adding commands to wp-cli

### ISSUE THREAD

**Issue by s-oliver** [woocommerce/woocommerce#28384]
We have automated testing for our plugin with woocommerce. For this purpose, we install and configure wordpress and woocommerce through the wp-cli, this no longer works with woocommerce 4.7.0

**Prerequisites (mark completed items with an [x]):**
- [X] I have have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

**Describe the bug**
We use the following steps to install and configure wordpress and woocommerce:
```
  - curl -O https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar
  - chmod +x wp-cli.phar && mv wp-cli.phar /usr/local/bin/wp
  - wp core install --url="$CONTAINER_BASE_URL" --title="Test" --admin_user="admin" --admin_password="xxx" --admin_email="support@test.com"
  - wp plugin install woocommerce-4.7.0.zip --activate
  - wp --user=admin option set woocommerce_currency EUR
  - wp wc product create --name="Test Product" --type=simple --sku=TEST --regular_price=20 --user=admin
  - wp wc tool run install_pages --user=admin
  - wp --user=admin wc shipping_zone create --name="Germany"
  - wp --user=admin wc shipping_zone_method create 1 --method_id=free_shipping
```

**Expected behavior**
Until now, this installed wordpress and the according woocommerce version. Woocommerce is installed, the product is created and the rest of the config ist set

**Actual behavior**
wp-cli reports, that `wc` is not a known subcommand:
```
Error: 'wc' is not a registered wp command. See 'wp help' for available commands.
Did you mean 'db'?
```

**Steps to reproduce the bug (We need to be able to reproduce the bug in order to fix it.)**
Steps to reproduce the bug:
1. Install latest wordpress from https://wordpress.org/latest.tar.gz
2. Install wp-cli from https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar
3. Install woocommerce 4.7.0
4. Run `wp wc help`

**Isolating the problem (mark completed items with an [x]):**
- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by rrennick** [comment]
@s-oliver Thanks for submitting the issue. I wasn't able to reproduce this in `master` with the latest WP 5.6 nightly.

```
$ wp wc product create --name="Test Product 2" --type=simple --sku=TEST2 --regular_price=20 --user=admin
Success: Created product 11505.
```

Did you confirm that WooCommerce was active when you ran `wp wc help`? If so, could you install the 4.8.0 RC and repeat your test?

**Comment by s-oliver** [comment]
Yes I made sure woocommerce was installed and active:

```
$ wp plugin list
+-------------------+----------+-----------+---------+
| name              | status   | update    | version |
+-------------------+----------+-----------+---------+
| akismet           | inactive | none      | 4.1.7   |
| hello             | inactive | none      | 1.7.2   |
| woocommerce-4.7.0 | active   | available | 4.7.0   |
+-------------------+----------+-----------+---------+

$ wp wc help
Error: 'wc' is not a registered wp command. See 'wp help' for available commands.
Did you mean 'db'?
```

On woocommerce 4.8.0RC1 everything works as expected.

I also tried it with 4.7.1 which was released yesterday, it works with that version again so I'm closing this issue
